src/main/java/org/broadinstitute/hellbender/tools/spark/pipelines/CountReadsSpark.java

```diff
@@ -28,7 +28,7 @@
 
     @Argument(doc = "uri for the output file: a local file path",
             shortName = StandardArgumentDefinitions.OUTPUT_SHORT_NAME, fullName = StandardArgumentDefinitions.OUTPUT_LONG_NAME,
-            optional = true)
+            optional = false)
```

Comment:
We are now forced to write the count to a file? I think this `-O` was optional for a good reason...